internal/verify/verify_test.go

```diff
@@ -27,40 +28,33 @@ import (
 
 func TestVerify(t *testing.T) {
 	eng := mkEngine(t)
-	runSuite := func(dir string) (*Result, error) {
+	runSuites := func(dir string) (*policyv1.TestResults, error) {
 		conf := Config{TestsDir: test.PathToDir(t, filepath.Join("verify", dir))}
 		return Verify(context.Background(), eng, conf)
 	}
 
 	t.Run("valid", func(t *testing.T) {
-		result, err := runSuite("valid")
+		result, err := runSuites("valid")
 		require.NoError(t, err)
-		require.Len(t, result.Suites, 4)
+		require.Len(t, result.Suites, 3)
 
 		for _, sr := range result.Suites {
 			t.Run(sr.File, func(t *testing.T) {
 				is := assert.New(t)
 				switch sr.File {
-				case "empty_test.yaml":
-					is.False(sr.Skipped)
-					is.Empty(sr.Principals)
```

Comment:
I removed this test case because while it technically passed, `sr.Failed` was `true`; an empty test suite is not actually valid, and we have coverage for the "invalid test" case below.